### ISSUE TITLE
Error during analysis with Java 11

### ISSUE THREAD

**Issue by nhenneaux** [spotbugs/sonar-findbugs#217]
## Issue Description

When analyzing with Java 11, the analysis fails while with Java 10 it works fine.

```
The following errors occurred during analysis:
Unable to get XClass for java/lang/Integer
  java.lang.UnsupportedOperationException
    At org.objectweb.asm.ClassVisitor.visitNestMemberExperimental(ClassVisitor.java:248)
    At org.objectweb.asm.ClassReader.accept(ClassReader.java:651)
    At edu.umd.cs.findbugs.asm.FBClassReader.accept(FBClassReader.java:44)
    At org.objectweb.asm.ClassReader.accept(ClassReader.java:391)
    At edu.umd.cs.findbugs.classfile.engine.ClassParserUsingASM.parse(ClassParserUsingASM.java:519)
    At edu.umd.cs.findbugs.classfile.engine.ClassParserUsingASM.parse(ClassParserUsingASM.java:703)
    At edu.umd.cs.findbugs.classfile.engine.ClassInfoAnalysisEngine.analyze(ClassInfoAnalysisEngine.java:79)
    At edu.umd.cs.findbugs.classfile.engine.ClassInfoAnalysisEngine.analyze(ClassInfoAnalysisEngine.java:38)
    At edu.umd.cs.findbugs.classfile.impl.AnalysisCache.getClassAnalysis(AnalysisCache.java:262)
    At edu.umd.cs.findbugs.ba.XFactory.getXClass(XFactory.java:687)
    At edu.umd.cs.findbugs.ba.AnalysisContext.setAppClassList(AnalysisContext.java:975)
    At edu.umd.cs.findbugs.FindBugs2.setAppClassList(FindBugs2.java:877)
    At edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:223)
    At org.sonar.plugins.findbugs.FindbugsExecutor$FindbugsTask.call(FindbugsExecutor.java:235)
    At java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    At java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
    At java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
    At java.base/java.lang.Thread.run(Thread.java:834)
Unable to get XClass for java/lang/StringBuilder
  java.lang.UnsupportedOperationException
    At org.objectweb.asm.ClassVisitor.visitNestMemberExperimental(ClassVisitor.java:248)
    At org.objectweb.asm.ClassReader.accept(ClassReader.java:651)
    At edu.umd.cs.findbugs.asm.FBClassReader.accept(FBClassReader.java:44)
    At org.objectweb.asm.ClassReader.accept(ClassReader.java:391)
    At edu.umd.cs.findbugs.classfile.engine.ClassParserUsingASM.parse(ClassParserUsingASM.java:519)
    At edu.umd.cs.findbugs.classfile.engine.ClassParserUsingASM.parse(ClassParserUsingASM.java:703)
    At edu.umd.cs.findbugs.classfile.engine.ClassInfoAnalysisEngine.analyze(ClassInfoAnalysisEngine.java:79)
    At edu.umd.cs.findbugs.classfile.engine.ClassInfoAnalysisEngine.analyze(ClassInfoAnalysisEngine.java:38)
    At edu.umd.cs.findbugs.classfile.impl.AnalysisCache.getClassAnalysis(AnalysisCache.java:262)
    At edu.umd.cs.findbugs.ba.XFactory.getXClass(XFactory.java:687)
    At edu.umd.cs.findbugs.ba.ch.Subtypes2.addInheritanceEdge(Subtypes2.java:1344)
    At edu.umd.cs.findbugs.ba.ch.Subtypes2.addSupertypeEdges(Subtypes2.java:1317)
    At edu.umd.cs.findbugs.ba.ch.Subtypes2.addClassAndGetClassVertex(Subtypes2.java:283)
    At edu.umd.cs.findbugs.ba.ch.Subtypes2.addClass(Subtypes2.java:252)
    At edu.umd.cs.findbugs.ba.AnalysisContext.setAppClassList(AnalysisContext.java:985)
    At edu.umd.cs.findbugs.FindBugs2.setAppClassList(FindBugs2.java:877)
    At edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:223)
    At org.sonar.plugins.findbugs.FindbugsExecutor$FindbugsTask.call(FindbugsExecutor.java:235)
    At java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    At java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
    At java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
    At java.base/java.lang.Thread.run(Thread.java:834)
(...)
Couldn't get class info for java/lang/Class
  java.lang.UnsupportedOperationException
    At org.objectweb.asm.ClassVisitor.visitNestMemberExperimental(ClassVisitor.java:248)
    At org.objectweb.asm.ClassReader.accept(ClassReader.java:651)
    At edu.umd.cs.findbugs.asm.FBClassReader.accept(FBClassReader.java:44)
    At org.objectweb.asm.ClassReader.accept(ClassReader.java:391)
    At edu.umd.cs.findbugs.classfile.engine.ClassParserUsingASM.parse(ClassParserUsingASM.java:519)
    At edu.umd.cs.findbugs.classfile.engine.ClassParserUsingASM.parse(ClassParserUsingASM.java:703)
    At edu.umd.cs.findbugs.classfile.engine.ClassInfoAnalysisEngine.analyze(ClassInfoAnalysisEngine.java:79)
    At edu.umd.cs.findbugs.classfile.engine.ClassInfoAnalysisEngine.analyze(ClassInfoAnalysisEngine.java:38)
    At edu.umd.cs.findbugs.classfile.impl.AnalysisCache.getClassAnalysis(AnalysisCache.java:262)
    At edu.umd.cs.findbugs.ba.XFactory.getXClass(XFactory.java:687)
    At edu.umd.cs.findbugs.ba.AnalysisContext.setAppClassList(AnalysisContext.java:975)
    At edu.umd.cs.findbugs.FindBugs2.setAppClassList(FindBugs2.java:877)
    At edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:223)
    At org.sonar.plugins.findbugs.FindbugsExecutor$FindbugsTask.call(FindbugsExecutor.java:235)
    At java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
    At java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
    At java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
    At java.base/java.lang.Thread.run(Thread.java:834)

```
## Environment
* Docker image from OpenJDK official repository https://hub.docker.com/_/openjdk/
[Docker OpenJDK 11 JRE slim](https://github.com/docker-library/openjdk/blob/894a19d0401349d39ced7764190ea6263bb17ba0/11/jdk/slim/Dockerfile)
* Maven 3.5.4
```
[INFO] SonarQube version: 6.7.1
[INFO] Default locale: "en", source code encoding: "UTF-8"
[INFO] Publish mode
[INFO] Load global settings
[INFO] Load global settings (done) | time=257ms

[INFO] User cache: /root/.sonar/cache
[INFO] Load plugins index
[INFO] Load plugins index (done) | time=57ms
[INFO] Download sonar-build-breaker-plugin-2.2.jar
[INFO] Download sonar-crowd-plugin-2.0.jar
[INFO] Download sonar-findbugs-plugin-3.8.0.jar
[INFO] Download sonar-flex-plugin-2.3.jar
[INFO] Download sonar-scm-git-plugin-1.4.1.1128.jar
[INFO] Download sonar-csharp-plugin-6.7.0.4267.jar
[INFO] Download sonar-javascript-plugin-4.1.0.6085.jar
[INFO] Download sonar-java-plugin-5.8.0.15699.jar
[INFO] Download sonar-php-plugin-2.12.0.2871.jar
[INFO] Download sonar-python-plugin-1.8.0.1496.jar
[INFO] Download sonar-typescript-plugin-1.2.0.1484.jar
```

**Comment by KengoTODA** [comment]
I think it's know issue solved by #211, we need to use [SpotBugs 3.1.7 that supports analyzing Java 11](https://github.com/spotbugs/spotbugs/blob/release-3.1/CHANGELOG.md#317---2018-09-12) but sonar-findbugs 3.8 depends on SpotBugs 3.1.6.

As a workaround, you can try [snapshot version of sonar-findbugs](https://oss.sonatype.org/content/repositories/snapshots/com/github/spotbugs/sonar-findbugs-plugin/3.9.0-SNAPSHOT/).

**Comment by nhenneaux** [comment]
Ok great ! Any idea when the release will be published?

**Comment by sabaka** [comment]
@KengoTODA , is there any approximate release date with spotbugs 3.1.7?

**Comment by KengoTODA** [comment]
solved by 3.9
